container-azure-go/go.mod

```diff
@@ -3,12 +3,12 @@ module ${PROJECT}
 go 1.18
 
 require (
-	github.com/pulumi/pulumi-azure-native-sdk/containerinstance v1.88.1
-	github.com/pulumi/pulumi-azure-native-sdk/containerregistry v1.88.1
-	github.com/pulumi/pulumi-azure-native-sdk/resources v1.88.1
+	github.com/pulumi/pulumi-azure-native-sdk/containerinstance/v2 v2.0.0
+	github.com/pulumi/pulumi-azure-native-sdk/containerregistry/v2 v2.0.0
+	github.com/pulumi/pulumi-azure-native-sdk/resources/v2 v2.0.0
 	github.com/pulumi/pulumi-random/sdk/v4 v4.8.2
 	github.com/pulumi/pulumi-docker/sdk/v4 v4.2.3
-	github.com/pulumi/pulumi/sdk/v3 v3.42.0
+	github.com/pulumi/pulumi/sdk/v3 v3.72.0
```

Comment:
This bumps to `v3.72.0` but `azure-go/go.mod` bumps to `v3.74.0`. Any reason for the inconsistency?